browser/main/Main.js

```diff
@@ -320,7 +326,23 @@ class Main extends React.Component {
 
   render() {
     const { config } = this.props
-
+    if (this.state.globRenderFlag !== 1) {
+      this.props.location.search = `?key=${this.state.currentNoteKey}`
+      this.props.data.showFullScreen = false
```

Comment:
This is strange, I don't think we can modify the `props` directly like this because `props` are read-only
https://reactjs.org/docs/components-and-props.html#props-are-read-only